tests/artemis-test-support/src/main/java/org/apache/activemq/transport/amqp/client/AmqpConnection.java

```diff
@@ -295,31 +294,6 @@ public void run() {
       return session;
    }
 
-   //----- Access to low level IO for specific test cases -------------------//
-
-   public void sendRawBytes(final byte[] rawData) throws Exception {
```

Comment:
You removed this because it was dead code?

Reply:
Yep!